source/clear-linux/tutorials/telemetry-e2e.rst

```diff
@@ -0,0 +1,294 @@
+.. _telemetry-e2e:
+
+
+|CL-ATTR| includes a telemetry and analytics solution (also known as telemetrics) as part of the OS, which records events of interest and reports them back to the development team using the telemetrics client daemons.
+End users can enable or disable the telemetry client component of |CL| and also redirect where records are sent if they wish to collect records for themselves by using their own telemetry backend server.
+
+This tutorial walks you through creating a telemetry enabled app on your local |CL| machine. The tutorial will show you how to send your own custom reports to any telemetry backend server.
+
+.. note::
+
+   The telemetrics functionality adheres to `Intel privacy policies`_ regarding the collection and use of :abbr:`PII (Personally Identifiable Information)` and is Open Source. Specifically, no intentionally identifiable information about the user or system owner is collected.
+
+Prerequisites
+=============
+
+For this tutorial, start with a clean installation of |CL| on a new system
```

Comment:
This really isn't a prerequisite.  The only prereq is that they have the telemetrics bundle installed on their system and they can check to see if the bundle is installed using swupd bundle-list.

Reply:
I can clean this up -- make it clear that the user won't be able to access their records on the default server.  The next paragraph points people back to the telemetry config guide page to show how to set up for local retention.